hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -4081,6 +4081,24 @@ public static boolean isAclEnabled(Configuration conf) {
   public static final long DEFAULT_FEDERATION_AMRMPROXY_SUBCLUSTER_TIMEOUT =
       60000; // one minute
 
+  public static final String FEDERATION_OVERRIDE_MAX_CLUSTER_CAPABILITY =
+          FEDERATION_PREFIX + "override.maximum.cluster.capability.enabled";
```

Comment:
We need to pay attention to indentation.

Reply:
We need to add some comments to explain the purpose of the variables.

Reply:
Revised and added.